gammapy/estimators/utils.py

```diff
@@ -6,16 +6,18 @@
 from astropy.table import Table
 from gammapy.datasets import SpectrumDataset, SpectrumDatasetOnOff
 from gammapy.datasets.map import MapEvaluator
-from gammapy.maps import MapAxis, TimeMapAxis, WcsNDMap
+from gammapy.maps import Map, MapAxis, TimeMapAxis, WcsNDMap
 from gammapy.modeling.models import (
     ConstantFluxSpatialModel,
     PowerLawSpectralModel,
     SkyModel,
 )
 from gammapy.stats import compute_flux_doubling, compute_fpp, compute_fvar
+from gammapy.stats.utils import ts_to_sigma
 from .map.core import FluxMaps
 
 __all__ = [
+    "get_joint_significance_maps",
```

Comment:
Is it better to call it `get_joint_TS_maps` rather than `get_joint_significance_maps`?
Because that is what we show mostly. The Estimators return `sqrt_ts` maps rather than `significance` maps.